java/ql/src/experimental/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -0,0 +1,55 @@
+/**
+ * @name openStream called on URLs created from remote source
+ * @description Calling openStream on URLs created from remote source
+ * can lead to local file disclosure.
+ * @kind path-problem
+ */
```

Comment:
This is missing an `@id` tag, isn't it?
Or is this not required for experimental queries?